Branch2018/test1/test.sql

```diff
@@ -0,0 +1 @@
+SELECT  HOST_NAME() AS 'host_name()',
```

Comment:
there is not necessary to upload files without use.

Reply:
ok, ready